### DISCUSSION TITLE
self-hostend Instance and push to Android/IOs

### DISCUSSION THREAD

**Discussion by steschuser** [mastodon/mastodon#18694] [Q&A]
We have a self-hosted instance and everything is working fine.

However, we don't receive notifications in the Mastodon app while Browser Notifications work fine.
Is there a way to enable this? I did not find antyhing relevant in the documentation.

best regards,
Steffen

**Comment by Gargron** [comment]
Both browsers and apps use Web Push for notifications. There should be no difference.

**Reply by nightpool** [reply]
Possibly the user is talking about desktop notifications, not push notifications

**Reply by drued** [reply]
I'm using Mastodon via Google Chrome on Windows 11 platform. After activating Desktop-Notifications in my Chrome browser, I receive the messages as configured on my desktop. On my Android smartphone, I get no push notifications. Whether via Chrome browser nor via the original Mastodon App.  The question is, what can I do to make push notifications working on mobile devices?
